server/szurubooru/api/info_api.py

```diff
@@ -64,3 +64,28 @@ def get_info(ctx: rest.Context, _params: Dict[str, str] = {}) -> rest.Response:
         )
         ret["featuringTime"] = post_feature.time if post_feature else None
     return ret
+
+
+@rest.routes.get(r"/manifest\.json")
```

Comment:
This might be entirely a non-issue, but is it possible that a request would come to this endpoint with `Accept: application/json`? (because I see that all the accept:json requests not to /api get denied in the nginx conf)

Reply:
I've redefined the location of the manifest file to be `/api/manifest.json` instead of `/manifest.json` (see [here](https://github.com/rr-/szurubooru/blob/17152f68a4b8594c8c72874936e2ed7028846d10/server/szurubooru/api/opengraph_api.py#L80)) to accommodate this.